taskvine/src/worker/vine_worker.c

```diff
@@ -208,6 +212,112 @@ __attribute__((format(printf, 2, 3))) void send_message(struct link *l, const ch
 	va_end(va);
 }
 
+/*
```

Comment:
Bump the number in `vine_protocol.h` since you are changing the on-the-wire protocol.